### PR TITLE
Genome wide annotations for Bovine, Canine, Zebrafish, Chicken, Rat, and Pig.

### PR DESCRIPTION
**Pull request by kurtwheeler**
* [X] I have read the guidelines above.
* [X] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

I based each of these recipes off of https://github.com/bioconda/bioconda-recipes/tree/master/recipes/bioconductor-org.hs.eg.db.

### CODE REVIEW

**Comment by dhimmel** [comment]
@bgruening @croth1 @druvus (or other maintainers): have you had a chance to look at this PR?

Would love to see these packages on conda and the bioconda channel seems like the right place.

File: recipes/bioconductor-org.bt.eg.db/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  md5: 3bece56d5a70cc7ca27a74857fed424c
+build:
+  number: 1
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - 'bioconductor-annotationdbi >=1.33.10'
+    - r
```
Comment (bgruening) [inline recipes/bioconductor-org.bt.eg.db/meta.yaml]: can we use `r-base` here?

File: recipes/bioconductor-org.cf.eg.db/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  md5: b731919179111906844470869c4e5ce6
+build:
+  number: 1
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - 'bioconductor-annotationdbi >=1.33.10'
+    - r
```
Comment (bgruening) [inline recipes/bioconductor-org.cf.eg.db/meta.yaml]: r-base

File: recipes/bioconductor-org.cf.eg.db/build.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+# R refuses to build packages that mark themselves as
+# "Priority: Recommended"
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
+#
+$R CMD INSTALL --build .
+#
+# # Add more build steps here, if they are necessary.
+#
+# See
```
Comment (bgruening) [inline recipes/bioconductor-org.cf.eg.db/build.sh]: This can be cleaned up. These comments are not needed.

**Comment by bgruening** [comment]
@kurtwheeler @dhimmel sorry, missed this one. I added a few comments inline.

**Comment by kurtwheeler** [comment]
@bgruening I've made the changes you suggested.

File: recipes/bioconductor-org.bt.eg.db/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: bioconductor-org.bt.eg.db
+  version: 3.4.0
+source:
+  fn: org.Bt.eg.db_3.4.0.tar.gz
+  url: http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Bt.eg.db_3.4.0.tar.gz
+  md5: 3bece56d5a70cc7ca27a74857fed424c
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/bioconductor-org.bt.eg.db/meta.yaml]: You can start this one with 0.

File: recipes/bioconductor-org.cf.eg.db/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: bioconductor-org.cf.eg.db
+  version: 3.4.0
+source:
+  fn: org.Cf.eg.db_3.4.0.tar.gz
+  url: http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Cf.eg.db_3.4.0.tar.gz
+  md5: b731919179111906844470869c4e5ce6
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/bioconductor-org.cf.eg.db/meta.yaml]: This one as well.

File: recipes/bioconductor-org.dr.eg.db/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: bioconductor-org.dr.eg.db
+  version: 3.4.0
+source:
+  fn: org.Dr.eg.db_3.4.0.tar.gz
+  url: http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Dr.eg.db_3.4.0.tar.gz
+  md5: 3e2b48c49db198076f7c97a7ddeda6bb
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/bioconductor-org.dr.eg.db/meta.yaml]: And this one :)

File: recipes/bioconductor-org.rn.eg.db/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: bioconductor-org.rn.eg.db
+  version: 3.4.0
+source:
+  fn: org.Rn.eg.db_3.4.0.tar.gz
+  url: http://bioconductor.org/packages/release/data/annotation/src/contrib/org.Rn.eg.db_3.4.0.tar.gz
+  md5: 7ebc8462e8ffba6b798086a7f6885d64
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/bioconductor-org.rn.eg.db/meta.yaml]: all of them :)

**Comment by kurtwheeler** [comment]
@bgruening I've made those changes as well.

**Comment by kurtwheeler** [comment]
@bgruening Is there anything else holding this up? I'd really like to get these into the bioconda channel.

**Comment by bgruening** [comment]
Please fix with the next iteration your bash script. It should startup with the shebang and not an empty line.

**Comment by kurtwheeler** [comment]
Whoopsie, I shouldn't have left those in. That's done though. Anything else?

**Comment by bgruening** [comment]
@kurtwheeler we have a new problem. BioiC was released in a new version so your old URLs do not work. This happends in the last days. Sorry. Can you change the URLs, fix the build.sh files and submit again a PR. You don't need to increase the build number, as the packages were not pushed :(

**Comment by kurtwheeler** [comment]
@bgruening I've been working on making those changes but the new versions require AnnotationDbi >=1.37.4 so I started a branch to upgrade it [here](https://github.com/kurtwheeler/bioconda-recipes/tree/update-annotationdbi-version) but my [build is failing](https://travis-ci.org/kurtwheeler/bioconda-recipes/jobs/228043917) because of an error message:
```
bioconda-utils lint: error: argument --git-range: expected at least one argument
```
That seems unrelated to anything I did. Do you know what might have caused that issue or how I should go about fixing it?

**Comment by bgruening** [comment]
Can you create a WIP PR please. this is some weirdness with out build env.

**Comment by kurtwheeler** [comment]
Sure.

**Comment by kurtwheeler** [comment]
PR is open here: https://github.com/bioconda/bioconda-recipes/pull/4581

**Comment by kurtwheeler** [comment]
Wait this got merged already? I thought there was a dependency issue I was still working on resolving due to the new version of BioiC.

**Comment by bgruening** [comment]
it was green so it was building correctly, do you see a package?

**Comment by kurtwheeler** [comment]
Nope, it doesn't turn up in a search on Anaconda.org

**Comment by johanneskoester** [comment]
I guess when the PR was built when the URLs were still working. After it has been merged into master and bioconda-utils tried to build it again for uploading, the URLs were gone and it could not build them. That's why the package do not appear on anaconda.org. After PR #4581 has been merged, you can create a new PR with the updated recipes.

**Comment by johanneskoester** [comment]
@bgruening, didn't we have backup URLs for bioconductor via Galaxy?

**Comment by kurtwheeler** [comment]
Ok I've created a new PR with the recipes here: https://github.com/bioconda/bioconda-recipes/pull/4597

**Comment by johanneskoester** [comment]
Thanks!

**Comment by bgruening** [comment]
@johanneskoester yes we have backups under https://github.com/galaxyproject/cargo-port and @daler was working on getting them included in the recipes by default: https://github.com/bioconda/bioconda-recipes/pull/3713

**Pull request merged**